File: for-local-test/268.missing-number.java

class Solution {
    public int missingNumber(int[] nums) {
        int n = nums.length;
        int sum = n * (n + 1) / 2;
        for(int num : nums){
            sum -= num;
        }
        return sum;
    }

    public static void main(String[] args) {
        Solution solution = new Solution();
        int[] nums = {3, 0, 1};
        int missingNumber = solution.missingNumber(nums);
        System.out.println("Missing number: " + missingNumber); // Output: Missing number: 2
    }
}

File: for-local-test/python/268.missing-number.py
class Solution:
    def missingNumber(self, nums: list[int]) -> int:
        n = len(nums)
        total_sum = n * (n + 1) // 2
        actual_sum = sum(nums)
        return total_sum - actual_sum

if __name__ == "__main__":
    solution = Solution()
    nums = [3, 0, 1]
    missing_number = solution.missingNumber(nums)
    print(f"Missing number: {missing_number}")  # Output: Missing number: 2P